.github/workflows/main.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         with:
           api: 29
           tag: google_apis
-          cmd: ./gradlew :publisher-sdk-tests:connectedCheck
+          cmd: ./gradlew :publisher-sdk-tests:connectedCheck --info
```

Comment:
Yep that is a massive amount of logs :D 
But I've seen recently CI jobs that were never finished. Using the --info shows all tests that are executed, whereas without, only the failing/ignored tests are printed.
So this is a way to see which tests may loop indefinitely.

As explained in the commit message, I tried to find an alternative to show such details like https://github.com/radarsh/gradle-test-logger-plugin. Unfortunately, the Android test logging is not configurable and is not link to anything in Gradle (except the logger).

This may only be temporary, as I discovered that the freeze-tests killer was deactivated: https://github.com/criteo/android-publisher-sdk/pull/26/commits/7a960b0d25ae037b7e04c259f3d8f0de06a26dd9
If after few times, nothing hang, I may remove this.